openfl/interface/workspace.py

```diff
@@ -232,6 +234,11 @@ def certify():
     echo('1.  Create Root CA')
     echo('1.1 Create Directories')
 
+    if cert_path:
+        CERT_PATH = Path(cert_path).absolute()
```

Comment:
This is a local variable, not a constant. The name should use lowercase letters to avoid misunderstanding.